gallery/src/components/NxStatefulIconDropdown/NxStatefulIconDropdownPage.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode, NxList, NxP, NxTextLink } from '@sonatype/react-shared-components';
+
+import {GalleryDescriptionTile, GalleryExampleTile} from '../../gallery-components/GalleryTiles';
+
+import NxStatefulIconDropdownExample from './NxStatefulIconDropdownExample';
+
+const nxStatefulIconDropdownExampleCode = require('./NxStatefulIconDropdownExample?raw');
+
+const NxStatefulIconDropdownPage = () =>
+  <>
+    <NxP>
+      <NxCode>NxStatefulIconDropdown</NxCode> utilizes the same Props as <NxCode>NxIconDropdown</NxCode> with
+      the exceptions of:
+    </NxP>
+    <NxList bulleted>
+      <NxList.Item><NxCode>isOpen</NxCode></NxList.Item>
+      <NxList.Item><NxCode>onToggleCollapse</NxCode></NxList.Item>
+      <NxList.Item><NxCode>onCloseKeyDown</NxCode></NxList.Item>
+      <NxList.Item><NxCode>onCloseClick</NxCode></NxList.Item>
+    </NxList>
```

Comment:
Why isn't this in the tile

Reply:
> Why isn't this in the tile

Good question: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/dd1395ef9ebbc154bf8a89094e455814d5bc5358